README.md

```
@@ -14,102 +14,113 @@ Alpha One Labs is an education platform designed to facilitate both learning and
 
-- Create Database:
+Follow these step-by-step instructions to set up your Cloudflare D1 database:
+
+1. **Create the Database:**
 
 ```bash
 wrangler d1 create education_db
 ```
 
-- Add the generated database_id to your wrangler.toml:
+2. **Add configuration to `wrangler.toml`:**
```

markdownlint-cli2 (0.22.0)

[warning] Ordered list item prefix
Expected: 1; Actual: 2; Style: 1/1/1

(MD029, ol-prefix)